src/azure-cli-core/azure/cli/core/extension/operations.py

```diff
@@ -167,6 +169,75 @@ def _add_whl_ext(cli_ctx, source, ext_sha256=None, pip_extra_index_urls=None, pi
     return extension_name
 
 
+def _install_deps_for_rdbms_connect():  # pylint: disable=too-many-statements
```

Comment:
could we name it as `_install_deps_for_psycopg2`? Then other extensions could reuse it.

Reply:
Make sense. 

In the future, we may even want a more generic method to support more packages that require extra system dependencies.

Reply:
Updated.